### DISCUSSION TITLE
How to toggle navbar hide show?

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#4479] [Q&A]
<img width="240" alt="image" src="https://github.com/mantinedev/mantine/assets/124666577/84032052-171f-4384-aae5-373a6920a7df">

```jsx
navbar={<Navbar   hiddenBreakpoint={'xl'} />}
```

Is my computer bigger than xl, so it's impossible for my computer to hide the sidebar?

**Comment by ItaiAxelrad** [comment]
You could change your breakpoints [https://mantine.dev/styles/responsive/#configure-breakpoints](url)

```js
theme={{
        breakpoints: {
          xs: '30em',
          sm: '48em',
          md: '64em',
          lg: '74em',
          xl: '90em', // <-whatever you want here
        },
      }}
```

Navbar also has a `hidden` prop that you could control with useState [https://mantine.dev/core/app-shell/#hidden-prop](url)

```js
const [opened, setOpened] = useState(false);
<Navbar hidden={!opened} hiddenBreakpoint="xl">
  <Navbar.Section grow mt='md'>
    Some content
  </Navbar.Section>
</Navbar>
```

**Reply by vaynevayne** [reply]
hidden, It seems the priority is too low, he will only take effect within the breakPoint range, which is a bit counter-intuitive
else, hiddenBreakpoint  It doesn't seem to be useful. The navbar is hidden on the xs screen, so the user has to be given an expanded button to switch,
 so it is still back to using hidden for control. Therefore, it is recommended to remove hiddenBreakpoint 

 @rtivital
